src/site/_filters/remove-drafts.js

```diff
@@ -12,6 +12,10 @@ module.exports = function removeDrafts(topics, lang) {
   return topics.reduce((accumulator, topic) => {
     // Remove draft posts from a topic.
     const posts = topic.pathItems.filter((slug) => {
+      if (typeof slug === 'object' && slug.url) {
+        // This is an external link.
+        return true;
+      }
```

Comment:
@robdodson looks like we lost our previous comment history. You had asked me to refactor to something like this. Does this match what you're looking for?

Reply:
Previous comment: https://github.com/GoogleChrome/web.dev/pull/4019#discussion_r507841398